src/TestSuite/ConsoleIntegrationTestCase.php

```diff
@@ -252,8 +252,10 @@ protected function _makeRunner()
     {
         if ($this->_useCommandRunner) {
             $applicationClassName = Configure::read('App.namespace') . '\Application';
+            /** @var \Cake\Http\BaseApplication $applicationClass */
+            $applicationClass = new $applicationClassName([CONFIG]);
```

Comment:
Did you on purpose revert the array to string in `new $applicationClassName(CONFIG)`?

Reply:
No I am just an idiot :facepalm: